tests/Aquifer.Common.UnitTests/Aquifer.Common.UnitTests.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\src\Aquifer.Common\Aquifer.Common.csproj" />
+    <ProjectReference Include="..\..\src\Aquifer.Common\Aquifer.Common.csproj" />
```

Comment:
I accidentally had the Aquifer.Common.UnitTests project in the root folder and moved it to `tests`.